core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2Spec.scala

```diff
@@ -403,7 +405,7 @@ class BillingApiServiceV2Spec extends ApiServiceSpec with MockitoSugar {
     verify(services.samDAO).deleteResource(SamResourceTypeNames.billingProject, project.projectName.value, userInfo)
     verify(services.samDAO).deleteResource(SamResourceTypeNames.googleProject, project.googleProjectId.value, userInfo)
   }
-  it should "return 204 - without google project" in withEmptyDatabaseAndApiServices { services =>
+  it should "return 204 on delete - without google project" in withEmptyDatabaseAndApiServices { services =>
```

Comment:
see line 385, this is already in the context of DELETE

Reply:
I wanted to add that, because it made it hard to see with the test runner. (since it just shows the text within the class component. Should I change `it` to be what we show on line 385?